### ISSUE TITLE
Unable to load addon script from path

### ISSUE THREAD

**Issue by jruots** [edbeeching/godot_rl_agents#55]
Hi,

I'm trying to set up my custom environment after having successfully followed the instructions to train ball chase.
First issue that I encountered was that I was not sure what this part of the text in the instructions was referring to "we have implemented a small plugin which is located in the plugin/addons/godot_rl_agents directory. Copy the plugin directory to your addons directory", meaning where is that directory from where I should copy the addon? I tried to workaround this by copying the godot_rl_agents folder from the ball chase example to the addons/ folder in my project.

Godot now recognized the addon, but when trying to enable it, I got the following error message: "Unable to load addon script from path: 'res://addons/godot_rl_agents/godot_rl_agents.gd'. This might be due to a code error in that script.
Disabling the addon at 'res://addons/godot_rl_agents/plugin.cfg' to prevent further errors."

So I guess two questions:
1) Where is the original directory where the addon should be?
2) What's behind the error and how do I solve the problem?

**Details**
- I'm using Godot 3.5.1

**Comment by jruots** [comment]
Ok so I figured out that the plugin is here: https://github.com/edbeeching/godot_rl_agents_plugin
I downloaded it from there, but still the same error message.

**Comment by jruots** [comment]
Thanks for the quick reply!

With the directory, I was looking for it in site packages in the python virtual env, which I then couldn't find. But I found the other github repo and downloaded the addon from there.

I downloaded Godot 4.10 now, but now my game of pong is very broken 😅 will need to try and fix that first before I can try whether the original issue went away with upgrading from 3.5.1 --> 4.10
